.travis.yml

```diff
@@ -43,6 +43,7 @@ matrix:
       osx_image: xcode9
       sudo: required
       env: SWIFT_SNAPSHOT=swift-DEVELOPMENT-SNAPSHOT-2017-08-15-a
+      env: JAZZY_ELIGIBLE=true
```

Comment:
It's not necessary to define this env on the Package-Builder repo itself.  Rather, it belongs on the `.travis.yml` of each repo that we want to enable for `jazzy-doc` generation.